stylesheets/_hacks.scss

```diff
@@ -106,3 +106,8 @@ blockquote p {
 .plain {
   text-decoration: none;
 }
+
+
+.floating:before {
+  margin-right: 0!important; // XXX JAMES SAID THIS IS OKAY
```

Comment:
@johnthepink that was something we used to have to do in junction but I'm not sure if it is still needed anymore. I'm pretty sure it was added to combat whitespace issues which aren't an issue anymore. I'd like to look around the app but I feel pretty good we can add this, and remove the hack from junction

eventually flexbox 👍